### ISSUE TITLE
show placeholder when ng-disabled="true" and used multiple

### ISSUE THREAD

**Issue by dtagdev1** [angular-ui/ui-select#1156]
show placeholder when ng-disabled="true" and used multiple

help me !

**Comment by dtagdev1** [comment]
I trying to show placeholder when ng-disabled="true" and used multiple. can someone help me?

**Comment by mgleed** [comment]
@dtagdev1 Hey, did you find a workaround for this problem? I've been running into it too.

**Comment by jziggas** [comment]
This seemed to work for me:
```
.ui-select-multiple.ui-select-bootstrap > div > input.ui-select-search[disabled] {
    display: initial !important;
    cursor: not-allowed;
}
```

**Comment by dtagdev1** [comment]
@jziggas awesome

**Comment by user378230** [comment]
No plunkr provided to test against but I believe this is now resolved in master (currently v0.19.6)
